src/components/Search/index.tsx

```diff
@@ -113,8 +119,9 @@ const StyledHighlight = styled(Highlight)`
   }
 `
 
+//FIXME: Add a strict type for `hit` prop
 const PageHit =
-  (clickHandler) =>
+  (clickHandler: MouseEventHandler): React.FC<{ hit: Record<string, any> }> =>
```

Comment:
@pettinarip 

Please let me know if there is a better (& strict) type for the `hit` prop.

Reply:
I just checked it out and we need to define the component's props without `React.FC` (I was having a conflict).

Lets define the props in the next line as
```ts
({ hit }: { hit: Hit<Record<string, any>> }) => {
```
You can import `Hit` from `@algolia/client-search`